tests/test_provider_constellix.py

```diff
@@ -894,6 +922,7 @@ def test_apply_healthcheck(self):
                     '/domains/123123/records/A',
                     data={
                         'name': 'www.dynamic',
+                        'note': 'rule-order:0',
```

Comment:
Might make sense to leave note blank on one of these to test the case mentioned above.

Reply:
Indeed. I will add this as well.

Reply:
Didn't look close enough. These are Calls. The place where rule-order can be left out is up above in `tests/fixtures/constellix-records.json`